README.md

```diff
@@ -48,14 +48,26 @@ See the [API documentation](https://raster-footprint.readthedocs.io/) for availa
 
 ## Developing
 
-Clone and install in editable mode with the development optional dependencies:
+Clone and install in editable mode with the development optional dependencies using [uv](https://docs.astral.sh/uv/):
 
 ```shell
 git clone https://github.com/pjhartzell/raster-footprint
 cd raster-footprint
+uv pip install -e ".[dev,docs]"
```

Comment:
As mentioned in a different comment, I want to leverage `uv` as much as possible and remove all methods that use `pip`. This can be replaced with `uv sync --all-extras`.